### PR TITLE
Update yolo.py

### PR DESCRIPTION
**Pull request by paartheee**
fixing #51

### CODE REVIEW

File: motrackers/detectors/yolo.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def __init__(self, weights_path, configfile_path, labels_path, confidence_thresh
         object_names = load_labelsjson(labels_path)
 
         layer_names = self.net.getLayerNames()
-        if cv2.__version__ == '4.6.0':
+        if cv.__version__ == '4.6.0':
```
Comment (taekim027) [inline motrackers/detectors/yolo.py:26]: Could we also make this check for versions greater than or equal to `4.6.0`?
```suggestion
        if cv.__version__ >= '4.6.0':
```

**Comment by taekim027** [comment]
LGTM, thank you so much! 💯

**Review by adipandas — APPROVED** [review]

**Comment by adipandas** [comment]
@partheee, @taekim027 Thank you for the contribution.

**Comment by taekim027** [comment]
Sweet, thank you! Any chance this could be deployed as an updated version installable via `pip`?

**Comment by adipandas** [comment]
@taekim027 - your changes are deployed in ``v0.0.2``

**Pull request merged**